lib/scim_rails/config.rb

```diff
@@ -1,26 +1,32 @@
+# frozen_string_literal: true
+
 module ScimRails
   class << self
     def configure
       yield config
     end
 
     def config
-      @_config ||= Config.new
+      @config ||= Config.new
```

Comment:
Rubocop indicates that the memoized instance variable should match the method name